src/events.h

```diff
@@ -172,11 +173,20 @@ namespace events {
         // The selected construct type has changed.
         extern signal<void(ConstructType)> construct_selected_event;
 
+        // Show the construct preview on the 3D build grid.
```

Comment:
Please explain all the parameters, providing example usage if unclear. I should be able to know how to use the event in my own code just from the comment.

Same for the other events you've defined.